src/database/models.py

```diff
@@ -76,44 +79,72 @@ def __str__(self):
 
 
 class UserReview(models.Model):
-    review_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
-    organization_id = models.ForeignKey(
-        Organization, on_delete=models.CASCADE, to_field="organization_id"
-    )
-    user_email = models.ForeignKey(
-        User, on_delete=models.CASCADE, to_field="user_email"
-    )
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    organization = models.ForeignKey(Organization, on_delete=models.CASCADE)
+    user = models.ForeignKey(User, on_delete=models.SET_NULL, null=True, blank=True)
     rating = models.IntegerField()
     comment = models.TextField(blank=True, null=True)
     active = models.BooleanField(default=True)
     created_at = models.DateTimeField(auto_now_add=True)
     modified_at = models.DateTimeField(auto_now=True)
 
     def __str__(self):
-        return f"Review by {self.user.first_name} for {self.organization.organization_name}"
+        return f"Review by {self.user.get_full_name() if self.user else 'Anonymous'} for {self.organization.name}"
 
 
 # As it currently stands it is users sending eachother messages, which needs to change to users exchanging messages with organizations.
 class Message(models.Model):
-    message_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
-    sender_id = models.ForeignKey(
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    sender_user = models.ForeignKey(
         User,
         on_delete=models.CASCADE,
         related_name="sent_messages",
-        to_field="user_email",
+        null=True,
+        blank=True,
+    )
+    sender_organization = models.ForeignKey(
+        Organization,
+        on_delete=models.CASCADE,
+        related_name="org_sent_messages",
+        null=True,
+        blank=True,
     )
-    receiver_id = models.ForeignKey(
+    receiver_user = models.ForeignKey(
         User,
         on_delete=models.CASCADE,
         related_name="received_messages",
-        to_field="user_email",
+        null=True,
+        blank=True,
+    )
+    receiver_organization = models.ForeignKey(
+        Organization,
+        on_delete=models.CASCADE,
+        related_name="org_received_messages",
+        null=True,
+        blank=True,
     )
     message_body = models.TextField()
     active = models.BooleanField(default=True)
     created_at = models.DateTimeField(auto_now_add=True)
 
     def __str__(self):
-        return f"Message from {self.sender.first_name} to {self.receiver.first_name}"
+        sender = (
+            self.sender_user.get_full_name()
+            if self.sender_user
+            else (
+                self.sender_organization.name if self.sender_organization else "Unknown"
+            )
+        )
+        receiver = (
+            self.receiver_user.get_full_name()
+            if self.receiver_user
+            else (
+                self.receiver_organization.name
```

Comment:
I think this should be {self.receiver_organization.organization_name} because name is not a field in organization

Reply:
will solve now